catalog/dags/data_refresh/dag_factory.py

```diff
@@ -105,25 +89,18 @@ def create_data_refresh_dag(data_refresh: DataRefresh, external_dag_ids: Sequenc
         )
 
         # Get the final number of records in the API table after the refresh
-        after_record_count = PGExecuteQueryOperator(
-            task_id="get_after_record_count",
-            conn_id=OPENLEDGER_API_CONN_ID,
-            sql=count_sql,
-            handler=single_value,
-            return_last=True,
+        after_record_count = es.get_record_count_group_by_sources(
```

Comment:
Same note here, except that in the case where we don't have an index, the skipping of the delete index step makes this skip too. We can correct that with a trigger rule:

```suggestion
        after_record_count = es.get_record_count_group_by_sources.override(
            task_id="get_after_record_count",
            trigger_rule=TriggerRule.NONE_FAILED_OR_SKIPPED,
        )(
```

Reply:
Perhaps `NONE_FAILED` would be better because, locally, we skip the tasks related to Cloudwatch alarms, but it would still be useful to have the report.

Reply:
It looks like `NONE_FAILED_OR_SKIPPED` is actually deprecated, so I agree I think `NONE_FAILED` is best (editing the suggestion for it)